### ISSUE TITLE
Append does not track metadata-- returns copy of metadata of just one lightcurve

### ISSUE THREAD

**Issue by gully** [KeplerGO/lightkurve#204]
Append does not track metadata-- returns copy of metadata of just one lightcurve

```python
lcf = KeplerLightCurveFile.from_archive(6922244, quarter=4)
lcf2 = KeplerLightCurveFile.from_archive(6922244, quarter=5)

lc = lcf.SAP_FLUX
lc2 = lcf2.SAP_FLUX

lc.channel, lc2.channel
(31, 3)

new_lc = lc_a.append(lc_b)
new_lc.channel
31
'''

Is this behavior desired?

**Comment by barentsen** [comment]
The only alternative would be to set `channel` and similar metadata to `None`?

**Comment by gully** [comment]
Exactly-- it's misleading to report erroneous or incomplete metadata for aggregated lightcurves.

**Comment by barentsen** [comment]
I wonder if we should remove the `lc.append()` method and let the concatenation a.k.a stitching of multiple lightcurves be the responsibility of the `LightCurveCollection` class.

**Comment by mirca** [comment]
Would appending the metadata be a feasible solution?

**Comment by barentsen** [comment]
@mirca You mean the value of `lc.channel` would become the list `[31, 3]`?

**Comment by mirca** [comment]
@barentsen Yeah, but I like your idea on having it on ``LightCurveCollection``. Maybe ``lc.append()`` could return a ``LightCurveCollection`` object?

**Comment by johnnyzhang295** [comment]
`LightCurveCollection` was built with the intention of holding _different_ lightcurves together, e.g. a cluster of stars. Now there was a previous implementation of `LightCurveCollection` that was able to hold lightcurves with the same `targetid` that could _potentially_ be revived...

**Comment by barentsen** [comment]
Just before merging I made a small change to make sure `LightCurveCollection` could hold multiple lightcurves with the same `targetid`.  I was worried users will be confused if a collection can't hold any arbitrary combination of `LightCurve` objects.

**Comment by barentsen** [comment]
I suggest the following way forward for this issue:
* Combining lightcurves should always return a generic `LightCurve` object that does not contain metadata that is specific to e.g. an individual `KeplerLightCurve` (e.g. `channel` is not an attribute of `LightCurve`).
* The combination of lightcurves should be implemented in a robust `LichtCurveCollection.stitch()` method.  The `LightCurve.append()` method should either be removed, or be a light wrapper around `LichtCurveCollection.stitch()`.

@nksaunders Do you want to look into this as part of #309?

**Comment by gully** [comment]
I think I agree that returning a generic `LightCurve` object is the most prudent way forward.  A common use case may be stitching split K2 campaigns like 11-1 and 11-2, where virtually all of the common metadata is the same, but lost.  Should these have special handling?
